client/src/components/style.css.js

```diff
@@ -34,84 +54,39 @@ const ellipsis = {
 };
 
 const carousel = {
+  width: '936px',
```

Comment:
If you're setting the width here, what does `maxWidth: 100%` do? I'm curious...

Reply:
It looks to me it is redundant here, let me delete that